### ISSUE TITLE
Fishyscapes OoD Dataset download

### ISSUE THREAD

**Issue by zhuyifan1993** [shjung13/Standardized-max-logits#4]
Hello,

Nice work first. I am trying to run your evaluation code, while I have to download the Fishyscapes OoD Dataset and put them in your specified data directory. 
I tried to install bdlb package and download the dataset through `bdlb.load(benchmark="fishyscapes")`, while the resulted data format/directory is different from yours.
Could you provide your script of data preprocessing or point me out how you did it. Thanks a lot!

BR,
Yifan

**Comment by shjung13** [comment]
Hello Yifan, 

Thank you for your interest in our work!

In our work, we manually re-organized dataset files to have the same structure as Cityscapes.

The dataset structure is posted on the README.md file :)

FYI, we utilized the latest version of the Fishyscapes static dataset (I guess it was version 3).

Hope this answer is helpful to you!

Best,
Sanghun

**Comment by zhouhuan-hust** [comment]
Hello，I have some questions about the Fishyscapes static dataset. Why do many of the labels I downloaded from V3 not contain anomalies? Are all 30 images in V3 used for testing? Thanks a lot!

**Comment by shjung13** [comment]
Hello, Zhouhuan.

I cannot exactly remember the process but there was an additional process that merges anomalies on normal images (since the static dataset is a synthetic one).

You may want to mail to the Fishyscapes team directly for exact instructions.

Thank you
